templates/components/checkbox.build.hbs

```diff
@@ -1,6 +1,29 @@
 <template>
   <div v-if="description" class="help-block description">\{{ description }}</div>
 
+   <template v-if="selectAll">
+    <div class="checkbox checkbox-icon">
+      <input
+        type="checkbox"
+        :id="name + '-' + 'selectAll'"
+        :name="name"
+        v-model="checkValue"
+        value="Select All"
+        tabindex="-1"
+      >
+      <label v-on:click="checkAllHandler()">
```

Comment:
Comment not addressed https://github.com/Fliplet/fliplet-widget-form-builder/pull/451/files